src/osd/PrimaryLogPG.cc

```diff
@@ -8173,6 +8131,19 @@ void PrimaryLogPG::_make_clone(
   rmattr_maybe_cache(obc, t, SS_ATTR);
 }
 
+bool PrimaryLogPG::need_clone(OpContext *ctx) 
```

Comment:
This portion and two related ones should be in the first commit with the set_manifest change, I think.